cupy/cuda/cublas.py

```diff
@@ -320,3 +320,19 @@ def sgemmBatched(handle, transa, transb, m, n, k, alpha, Aarray, lda, Barray,
 def sdgmm(handle, mode, m, n, A, lda, x, incx, C, ldc):
     status = _cublas.cublasSdgmm(handle, mode, m, n, A, lda, x, incx, C, ldc)
     check_status(status)
+
+_cublas.cublasSgetrfBatched.argtypes = [Handle, _I, _P, _I, _P, _P, _I]
+
+
+def sgetrf(handle, n, A, lda, P, info, batchSize):
```

Comment:
According to the naming convention of `cupy.cublas`, CuPy's method name is made by removing `cublas` and `v2` from the corresponding cublas method name, and making the first character lowercase. So this method name should be `sgetrfBatched`.